roles/database/files/sql/idempotent/fworch-texts.sql

```diff
@@ -558,11 +558,11 @@ INSERT INTO txt VALUES ('group_search_path',    'German', 	'Suchpfad Gruppen');
 INSERT INTO txt VALUES ('group_search_path',    'English', 	'Group Search Path');
 INSERT INTO txt VALUES ('search_user',          'German', 	'Nutzer f&uuml;r Suche');
 INSERT INTO txt VALUES ('search_user',          'English', 	'Search User');
-INSERT INTO txt VALUES ('search_user_pwd',      'German', 	'Passwort Nutzer f&uuml;r Suche');
+INSERT INTO txt VALUES ('search_user_pwd',      'German', 	'Pwt Nutzer f&uuml;r Suche');
 INSERT INTO txt VALUES ('search_user_pwd',      'English', 	'Search User Pwd');
 INSERT INTO txt VALUES ('write_user',           'German', 	'Schreibender Nutzer');
 INSERT INTO txt VALUES ('write_user',           'English', 	'Write User');
-INSERT INTO txt VALUES ('write_user_pwd',       'German', 	'Passwort Schreibender Nutzer');
+INSERT INTO txt VALUES ('write_user_pwd',       'German', 	'Pwt Schreibender Nutzer');
```

Comment:
I do not really like Pwt (and Pwd) is there a good reason for abbreviating this at all?